### PR TITLE
User can view task

### PR DESCRIPTION
**Pull request by Boyul-Kim**
2cf10ed4693c2b7c1ad149c917510a17

### CODE REVIEW

File: client/pages/task.jsx [hunk 1]
```diff
@@ -1,50 +1,81 @@
 import React from 'react';
 
 export default class Task extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      task: []
```
Comment (thebearingedge) [inline client/pages/task.jsx]: Arrays are usually for holding a list.

File: server/index.js [hunk 2]
```diff
@@ -145,6 +145,41 @@ app.post('/api/tasks/create', (req, res, next) => {
...
+    join "statuses" using ("statusId")
+    join "users" using ("userId")
+    where "taskId" = ($1)
+  `;
+
+  const param = [taskId];
+
+  db.query(sql, param)
+    .then(result => {
+      res.status(200).json(result.rows);
```
Comment (thebearingedge) [inline server/index.js]: If a client asks for one thing, then you need to only send them one thing or throw a 404

File: server/public/styles.css [hunk 3]
```diff
@@ -57,3 +57,21 @@
 .time-width {
   width: 9rem;
 }
+
+.viewTask {
```
Comment (thebearingedge) [inline server/public/styles.css]: Please be consistent with your CSS casing. We're using `kebab-case`

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please see my other notes, but otherwise try to:
- fix the bottom padding on your inputs. i have no idea why the text is not vertically centered in them.
- remove the padding around your side-by-side inputs so that they line up with the left and right of the other inputs.

**Comment by Boyul-Kim** [comment]
11a91bbeacd19df12c8cb0aff8074292

File: server/index.js [hunk 4]
```diff
@@ -145,6 +145,41 @@ app.post('/api/tasks/create', (req, res, next) => {
...
+    join "users" using ("userId")
+    where "taskId" = ($1)
+  `;
+
+  const param = [taskId];
+
+  db.query(sql, param)
+    .then(result => {
+      res.status(200).json(result.rows[0]);
+    })
```
Comment (thebearingedge) [inline server/index.js]: Be sure to throw your `404` here if it's not found. Otherwise, you are approved to merge.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Pull request merged**